src/routes/PaymentChoicePage.tsx

```diff
@@ -48,8 +48,11 @@ export default function PaymentChoicePage() {
 
   React.useEffect(() => {
     if (
-      (getSessionItem(SessionItems.transaction) as Transaction | undefined)
-        ?.transactionId
+      (
+        getSessionItem(SessionItems.transaction) as
```

Comment:
same speech reported previously 🙂
```  
pipe(
      getSessionItem(SessionItems.transaction),
      NewTransactionResponse.decode,
      E.fold(
        () => undefined,
        (transaction) => transaction.transactionId
      )
    )
  )
);
```

Reply:
Removed this old and unused check on this page